jdaviz/configs/imviz/helper.py

```diff
@@ -267,7 +267,7 @@ def get_aperture_photometry_results(self):
             Photometry results if available or `None` otherwise.
 
         """
-        return self.plugins['Imviz Simple Aperture Photometry']._obj.export_table()
+        return self.plugins['Aperture Photometry']._obj.export_table()
```

Comment:
I can look into this next week if its something we would want.  Again, it only affects show and open_in_tray methods, but if people use the API just to programmatically show the plugin, then it might be a good idea to properly deprecate.

Reply:
ok, this isn't easy, but is probably possible.  Do we think its necessary and worth the extra code infrastructure?

Reply:
Well, thanks for trying. In that case, maybe we can put an entry maybe in Known Issues or something? For that one person who happen to use the old public API lol...